src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,206 @@
+    )
+  }
+
+  public concat(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.CONCAT'),
+      this.interpreter.arithmeticHelper.concat
+    )
+  }
+
+  public divide(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.DIVIDE'),
+      this.interpreter.arithmeticHelper.divide
+    )
+  }
+
+  public eq(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.EQ'),
+        this.interpreter.arithmeticHelper.eq
+    )
+  }
+
+  public gt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GT'),
+        this.interpreter.arithmeticHelper.gt
+    )
+  }
+
+  public gte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.GTE'),
+        this.interpreter.arithmeticHelper.geq
+    )
+  }
+
+  public lt(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LT'),
+        this.interpreter.arithmeticHelper.lt
+    )
+  }
+
+  public lte(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.LTE'),
+        this.interpreter.arithmeticHelper.leq
+    )
+  }
+
+  public minus(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MINUS'),
+      this.interpreter.arithmeticHelper.subtract
+    )
+  }
+
+  public multiply(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.MULTIPLY'),
+      this.interpreter.arithmeticHelper.multiply
+    )
+  }
+
+  public ne(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.NE'),
+      this.interpreter.arithmeticHelper.neq
+    )
+  }
+
+  public pow(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.POW'),
+      this.interpreter.arithmeticHelper.pow
+    )
+  }
+
+  public uminus(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.UMINUS'),
+      this.interpreter.arithmeticHelper.unaryMinus
+    )
+  }
+
+  //eslint-disable-next-line @typescript-eslint/camelcase
+  public unary_percent(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
```

Comment:
Does it have to be an exception in the naming convention? I think that the compatiblity with GS and FORMULA.JS is handled at the metadata key / language level?